include/detect_breakends/junction_detection.hpp

```diff
@@ -22,6 +22,9 @@
  *                                                           1: hierarchical_clustering,
  *                                                           2: self-balancing_binary_tree,
  *                                                           3: candidate_selection_based_on_voting)
+ * \param refinement_method method for refining breakends (0: no_refinement,
+ *                                                         1: sViper,
+ *                                                         2: prioritzation_based_on_rank_aggregation)
```

Comment:
Prioritization spelling

Reply:
I'm not quite sure what you mean with the prioritization based on rank aggregation part. If this adapted from Vaquita, then it doesn't belong in refining breakpoints.

The prioritization used in Vaquita is more to "rank" the variants by how important they are. This also has the effect that the variants which are considered "more important" are more likely to be true variants. However, rank aggregation does not do anything to the existing breakpoints. It's sort of a "final step" in the pipeline.

For now, I would think that the only breakend refinement we have is via sViper.

Reply:
Yes you are right, the `prioritzation_based_on_rank_aggregation` is probably wrong here. I spoke to david and we have still 2 different refining methods. The other is a method from SVirl.
On the other side, I think we want to able and enable the refinement, as it also depends on the input. sViper only works if we have short and long reads. The method from sVirl works with long reads, and we have none for short reads only.

We could remove this parameter later and could do this decision automaticaly but for now it is easyer for us and we are able to add more if there are more methods.